README.md

```diff
@@ -1,4 +1,4 @@
-# Acorn
+# <img src="https://raw.githubusercontent.com/acornjs/acorn/refs/heads/master/logo.svg" alt="Acorn Logo" width="100"> Acorn
```

Comment:
Markdown syntax allows to use relative URLs (just "logo.svg"), does the same not work with img?

Reply:
I don't see how that answers the question. If anything, relative URL is more reliable and will work correctly in eg local editors if/when logo is being worked on.

Reply:
one case of this, if it's shallow by external app like npmjs.com.

But ofc we can use relative link it's will work.